### PR TITLE
aireplay-ng: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Use range syntax for placeholders and fix description.

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages/common/aireplay-ng.md [hunk 1]
```diff
@@ -4,6 +4,6 @@
 > Part of `aircrack-ng`.
 > More information: <https://www.aircrack-ng.org/doku.php?id=aireplay-ng>.
 
-- Send a specific number of disassociate packets given an access point's MAC address, a client's MAC address and an interface:
+- Send a specific number of disassociate packets with an access point's MAC address, a client's MAC address and an interface:
```
Comment (navarroaxel) [inline pages/common/aireplay-ng.md:7]: I think this example should be split, because `infinity` is not a `specific number`.

Comment (EmilySeville7cfg) [reply pages/common/aireplay-ng.md:7]: Yeah, `infinity` is. But the argument passed is concrete. 😄 `infinity` is a kind of abstraction over the highest possible number accepted by some program.

Comment (navarroaxel) [reply pages/common/aireplay-ng.md:7]: but with `infinity` you can explain how to stop the command with Ctrl+C

Comment (EmilySeville7cfg) [reply pages/common/aireplay-ng.md:7]: Sorry, but I didn't get your point: how placeholder relates to interrupting the command? 🤔 Can you provide more details about what u wanna tell me?

Comment (navarroaxel) [reply pages/common/aireplay-ng.md:7]: With a specific number the command exits when the loop completes. with infinity I think the behavior is similar to `ping`, loops until Ctrl+C.

Comment (EmilySeville7cfg) [reply pages/common/aireplay-ng.md:7]: In other words u wanna say with any big number this commands is expected to work for a long time? 😄

Comment (EmilySeville7cfg) [reply pages/common/aireplay-ng.md:7]: PR was closed, this change was cancelled.

**Pull request closed without merging**